package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@luxury-presence/nestjs-plunk",
-  "version": "3.0.0",
+  "version": "3.0.1-0",
```

Comment:
issue: same as the JSON:API library, this change is breaking if we're using SemVer, and this should be `4.0.0-0` (or `-1` or `-alpha.1` ...etc)